masschallenge/django-accelerator
# [AC-9445] Add IndustryCluster model

## [AC-9445](https://masschallenge.atlassian.net/browse/AC-9445)

**Changes introduced**
- add the IndustryCluster model

**Testing**
- Checkout branch `AC-9445` and run `make migrate` on the accelerate repo,
- Confirm that the def of done according to [AC-9445](https://masschallenge.atlassian.net/browse/AC-9445) is met

[Sibling PR](https://github.com/masschallenge/accelerate/pull/3251)
